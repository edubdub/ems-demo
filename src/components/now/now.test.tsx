import * as React from 'react'
import { shallow } from 'enzyme'
import NowButton from './now'

it('renders without crashing', () => {
  expect(shallow(<NowButton onPress={() => null} />)).toMatchSnapshot()
})
