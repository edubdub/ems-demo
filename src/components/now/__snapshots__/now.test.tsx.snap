// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Loading
    onPress={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "class",
    "props": Object {
      "children": <span
        style={
                Object {
                        "color": "white",
                      }
        }
>
        Now
</span>,
      "disabled": false,
      "mini": false,
      "onClick": [Function],
      "onKeyPress": [Function],
      "secondary": false,
      "style": Object {
        "bottom": 40,
        "display": "block",
        "position": "fixed",
        "right": 40,
        "textAlign": "center",
      },
      "zDepth": 2,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": "Now",
        "style": Object {
          "color": "white",
        },
      },
      "ref": null,
      "rendered": "Now",
      "type": "span",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "class",
      "props": Object {
        "children": <span
          style={
                    Object {
                              "color": "white",
                            }
          }
>
          Now
</span>,
        "disabled": false,
        "mini": false,
        "onClick": [Function],
        "onKeyPress": [Function],
        "secondary": false,
        "style": Object {
          "bottom": 40,
          "display": "block",
          "position": "fixed",
          "right": 40,
          "textAlign": "center",
        },
        "zDepth": 2,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Now",
          "style": Object {
            "color": "white",
          },
        },
        "ref": null,
        "rendered": "Now",
        "type": "span",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
